### DISCUSSION TITLE
Add engagement counts on the timeline

### DISCUSSION THREAD

**Discussion by osmaa** [NickColley/semaphore#29] [New features and ideas]
If you care for a feature like this, I made it a couple of weeks ago for my own use. This moves the boost and fav counts from the single-toot details screen to the timeline. Sure, it's a bit controversial vs Mastodon defaults, but also something an increasing number of other clients are doing.

I may also fork your new repo and apply my own modifications on top of it, but even happier to have a maintained upstream!

https://github.com/osmaa/semaphore/commit/b7fc840e77c99f6e18bdfc0e139e6a1a4f5568be

**Comment by NickColley** [comment]
Just so I can visualize this can you include a screenshot when you get a few seconds? Sounds interesting

**Reply by osmaa** [reply]
sure thing:
![image](https://user-images.githubusercontent.com/1743200/214271329-b951ce63-3341-4e1f-80c0-e3a42ca6fbc3.png)

The counts do have their respective Wellness toggles, too. I will re-initialize my repo and re-apply the patches since trying to rebase made a complete mess..

**Comment by osmaa** [comment]
okay, my own repo cleaned up and a PR for you at https://github.com/NickColley/semaphore/pull/32

**Comment by NickColley** [comment]
> I've spent some time thinking about this and will have to reject the proposal.
> 
> I want to keep Semaphore more focused around the content and less what I'd call 'engagement indicators', as for me personally I've noticed that sort of design is what made Twitter very poor for my mental health.
> 
> I recognize you explicitly considered this as part of your work with the wellness section, but I feel uncomfortable having this on by default and want to avoid designing by feature flags (and aim to remove or reduce them over time).
> 
> Thank for you taking the time to contribute, the work is great so I'm sorry I had to say no this time.


https://github.com/NickColley/semaphore/pull/32#issuecomment-1465241335
